src/main/resources/application.yml

```diff
@@ -183,3 +183,4 @@ nakadi.features.defaultFeatures:
   REMOTE_TOKENINFO: true
   KPI_COLLECTION: true
   DISABLE_DB_WRITE_OPERATIONS: false
+  FORCE_SUBSCRIPTION_AUTHZ: true
```

Comment:
For local deployments I would not make this mandatory: it keeps things as simple as possible.